internal/auth0/connection/schema.go

```diff
@@ -503,19 +503,14 @@ var resourceSchema = map[string]*schema.Schema{
 				},
 
 				"set_user_root_attributes": {
-					Type:     schema.TypeString,
-					Optional: true,
-					Computed: true,
```

Comment:
If we keep this as computed we won't be able to actually remove it from the update so we can have it use default for WAAD conns.